app/assets/javascripts/users.js

```diff
@@ -17,4 +17,26 @@ $(function(){
 		}
 		$(target).removeClass("inactive");
 	});
-});
\ No newline at end of file
+});
+
+$(function(){
+	$("#user-tabs-nav a[data-target]").click(function() {
+		var target = this.dataset.target;
+		var navLinks = $("#user-tabs-nav a[data-target]");
+		for (var i = 0; i < navLinks.length; i++) {
+			$(navLinks[i]).addClass("inactive");
+		}
+		$("a[data-target=" + target + "]").removeClass("inactive");
+
+		// update content
+		var target = "#user-" + this.dataset.target;
+		var contentDivs = $("#user-tabs-content > div");
+		for (var i = 0; i < contentDivs.length; i++) {
+			$(contentDivs[i]).addClass("inactive");
+		}
+		$(target).removeClass("inactive");
+	});
+});
```

Comment:
This is for the users index page (for admins)
Used the structure from the user show page
two columns: active users    and  users to be approved